quesma/jsonprocessor/ingest.go

```diff
@@ -0,0 +1,43 @@
+// Copyright Quesma, licensed under the Elastic License 2.0.
```

Comment:
The existing `quesma/jsonprocessor/json_processor.go` file is also used only for ingest, so maybe both of them should live in a single file.

However, I'm not sure if there should be a separate package`jsonprocessor (due to a risk of introducing circular dependencies later)

Reply:
+1